### PR TITLE
WIP: always downloading a fixed buffalo CLI version

### PR DESCRIPTION
**Pull request by arschles**
As of this writing, latest is v0.12.3

### CODE REVIEW

File: scripts/get_buffalo.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+BUFFALO_URL="https://github.com/gobuffalo/buffalo/releases/download/v0.12.3/buffalo_0.12.3_linux_amd64.tar.gz"
+TAR_GZ="buffalo_0.12.3_linux_amd64.tar.gz"
+
+curl -L -o ${TAR_GZ} ${BUFFALO_URL}
+tar -xvf ${TAR_GZ}
+mv buffalo-no-sqlite /usr/local/bin/buffalo
```
Comment (michalpristas) [inline scripts/get_buffalo.sh]: dont you need `+x`

Comment (arschles) [reply scripts/get_buffalo.sh]: I think it should already be executable in the tarball that gets downloaded, but no harm in doing so!

**Comment by arschles** [comment]
good idea @robjloranger. change incoming

**Comment by arschles** [comment]
@robjloranger just updated. mind taking another look?

**Pull request merged**